dev-resources/template/2_lrs.yml

```diff
@@ -1,44 +1,21 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: "SQL LRS and DB Init Script"
 Parameters:
-  # Networking
   VPCId:
-    Description: "VPC on which to run SQL LRS"
+    Description: VPC on which to run SQL LRS
     Type: AWS::EC2::VPC::Id
   DBSubnets:
     Description: Subnets on which the DB is running
     Type: List<AWS::EC2::Subnet::Id>
   # DB Details
-  DBName:
-    Description: PG Database name. Ignored if DBSnapshotIdentifier is provided
-    Type: String
-    Default: "lrsql_demo"
-  DBHost:
-    Description: Write hostname of PG db
-    Type: String
-  DBPort:
-    Description: Port of PG db
-    Type: String
-    Default: 3306
-  DBInstanceSG:
-    Type: AWS::EC2::SecurityGroup::Id
-    Description: DB instance security group
-  DBMasterUserName:
-    Description: Name of admin user
```

Comment:
I'm confused, I thought we were still going with the approach of making these optional

Reply:
Cliff clarified on slack we're OK with letting these go away